### PR TITLE
add unet-camvid-onnx-0001 description

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
add description of pre-trained on CamVid unet model for semantic segmentation

### CODE REVIEW

File: models/intel/unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+The model input is a blob that consists of a single image of "1x3x368x480" in BGR order. The pixel values are integers in the [0, 255] range.
+
+The model output for `unet-camvid-onnx-0001` is the per-pixel probabilities of each input pixel belonging to one of the 12 classes of the CamVid dataset.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| Type              | Semantic segmentation |
+| Source framework  | PyTorch\*             |
```
Comment (eizamaliev) [inline models/intel/unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md]: Is mParams and FLOPs values available for this model?

Comment (vladimir-dudnik) [reply models/intel/unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md]: ok, added

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+The model input is a blob that consists of a single image of "1x3x368x480" in BGR order. The pixel values are integers in the [0, 255] range.
+
+The model output for `unet-camvid-onnx-0001` is the per-pixel probabilities of each input pixel belonging to one of the 12 classes of the CamVid dataset.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| GFlops            | 260.1                 |
```
Comment (eizamaliev) [inline models/intel/unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md]:
```suggestion
| Type              | Semantic segmentation |
| GFlops            | 260.1                 |
```

Comment (vladimir-dudnik) [reply models/intel/unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md]: @eizamaliev what is the reason to specify model type here? Can you point to any other model descriptions which has specified type in this table?

Comment (eizamaliev) [reply models/intel/unet-camvid-onnx-0001/description/unet-camvid-onnx-0001.md]: Ok, my bad - thought  it is public model.

**Pull request merged**